ws-nextjs-app/next.config.js

```diff
@@ -24,7 +24,7 @@ module.exports = {
   experimental: {
     externalDir: true,
   },
-  env: clientEnvVars,
+  env: { ...clientEnvVars, LOG_TO_CONSOLE: 'true' },
```

Comment:
Sets NextJS app to always log to console so that Lambda can push them to CloudWatch